simd/cpu_amd64.s

```diff
@@ -0,0 +1,55 @@
+/* This file is a copy of https://cs.opensource.google/go/x/sys/+/master:cpu/cpu_x86.s.
```

Comment:
In case their file changes in the future, it might be better to reference a specific version of it

```suggestion
/* This file is a copy of https://cs.opensource.google/go/x/sys/+/refs/tags/v0.20.0:cpu/cpu_x86.s.
```